src/consensus/params.h

```diff
@@ -24,15 +24,16 @@ enum DeploymentPos
     // When a bit goes back to being unused, it should be renamed to
     // DEPLOYMENT_UNASSIGNED_BIT_x .
     // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
-    DEPLOYMENT_CSV = 0, // bit 0 - deployment of BIP68, BIP112, and BIP113.
```

Comment:
Why bit 0 was initially assigned to BIP68, BIP112, and BIP113? Haven't we got thewm already activated by the time we introduced BIP135?

Reply:
yes, but we left it there because there's plenty of bits and it was previously specified as CSV

Reply:
ok.